modules/polygon/src/polygon-utils.d.ts

```diff
@@ -19,7 +19,7 @@ type SegmentVisitorFlat = (
 ) => void;
 
 // Segment visitor callback type for polygons defined with array of points.
-type SegmentVisitorPoints = (p1: NumberArray, p2: NumberArray, i1: number, i2: number) => void;
+export type SegmentVisitorPoints = (p1: NumberArray, p2: NumberArray, i1: number, i2: number) => void;
```

Comment:
Are we deliberately only exporting the points version, not the flat version?

Reply:
I needed this type for the `forEachSegment(SegmentVisitorPoints): void;` function so I exported it. The flat version is only used internally AFAIK